services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/permissions/HankeKayttajaServiceITest.kt

```diff
@@ -591,8 +592,8 @@ class HankeKayttajaServiceITest : DatabaseTest() {
             val kayttajat = hankeKayttajaRepository.findAll()
             assertThat(kayttajat).hasSize(4)
             assertThat(kayttajat).each { kayttaja ->
-                kayttaja.transform { it.etunimi }.isEqualTo("Teppo")
-                kayttaja.transform { it.sukunimi }.isEqualTo("Testihenkilö")
+                kayttaja.transform { it.etunimi }.isEqualTo(ApplicationFactory.TEPPO)
```

Comment:
Could be imported as `import fi.hel.haitaton.hanke.factory.ApplicationFactory.Companion.TEPPO` as has been done elsewhere.

Reply:
Importing the properties directly.